moveit_ros/integration_tests/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(moveit_ros_integration_tests)
```

Comment:
I think we should move away from the moveit_ros subfolders eventually. Why not just call this ``moveit_integration_tests`` and leave in root folder?